package.json

```diff
@@ -17,19 +17,22 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "lit-element": "^2.5.1",
-    "chalk": "^4.1.2"
+    "chalk": "^4.1.2",
+    "lit-element": "^2.5.1"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.3.0",
-    "@alaskaairux/webcorestylesheets": "^3.6.3",
+    "@alaskaairux/design-tokens": "^3.15.5",
+    "@aurodesignsystem/design-tokens": "^4.2.0",
```

Comment:
If the new tokens were added and the code updated to support the `--ds` namespace, why is the legacy library still installed?

Reply:
oh for this one I follow this hyperlink feedback, a need to keep the legacy tokens around for the time being.
https://github.com/AlaskaAirlines/auro-hyperlink/pull/185#discussion_r1370351762

FYI I did this for the other currently open PR's, so if this is not needed, then I should remove those as well in the other components.

Reply:
Oh... that may be a misunderstanding. We do not need to keep the package dependency if both tokens and WCSS is updated and there is no other internal dependency to the element that needs this. 

But there are external peer dependencies that need the legacy tokens, so all we need to do is keep that on the `./demo/index.html` page.